app/models/menu.rb

```diff
@@ -0,0 +1,18 @@
+class Menu < ApplicationRecord
+  has_many :menu_items, dependent: :destroy
+  has_many :orders
+
+  accepts_nested_attributes_for :menu_items, allow_destroy: true, reject_if: proc { |att| att['name'].blank? }
+
+  validates :menu_date, uniqueness: true
+
+  scope :for_today, -> { find_by(menu_date: Time.now.to_date) }
+
+  def for_today?
+    menu_date == Time.now.to_date
+  end
+
+  def available_for_ordering?
+    for_today? && menu_items.present? 
```

Comment:
this is already too much for a model. Menu doesn't really have to know this. This goes straight to service where this logic is needed